### ISSUE TITLE
reporters.filter is not a function

### ISSUE THREAD

**Issue by gajus** [webpro/knip#105]
New install, unsure what is broken, but getting error:

```
Analyzing workspace (packages/temporary-common)...
file:///Users/gajus/Developer/contra/gaia/node_modules/.pnpm/knip@2.2.2/node_modules/knip/dist/plugins/vitest/helpers.js:25
    return reporters ? reporters.filter(reporter => !builtInReporters.includes(reporter)) : [];
                                 ^

TypeError: reporters.filter is not a function
```

I added `console.log` to `getExternalReporters` to see what is the value of `reporters` and it prints:

```
{ reporters: 'verbose' }
```

Our configuration:

```ts
import { type KnipConfig } from 'knip';

const Knip: KnipConfig = {
  "ignoreWorkspaces": ["apps/*"],
  "workspaces": {
    "packages/*": {
      "ignore": [
        "**/*.d.ts"
      ],
      "entry": [
        "src/index.ts!"
      ],
      "project": [
        "src/**/*.ts!"
      ]
    },
  }
};

export default Knip;
```

**Comment by webpro** [comment]
Ah, `reporters` can be a string too, should be fixed in [v2.2.3](https://github.com/webpro/knip/releases/tag/2.2.3).

**Comment by webpro** [comment]
By the way, I can see this project has vitest and that configuration (e.g. `vitest.config.js`) has `{ reporters: 'verbose' }`
